swan-cern/values.yaml

```diff
@@ -174,6 +204,8 @@ swan:
           description: Access to hub Prometheus metrics
           scopes: ['read:metrics']
           services: [prometheus-service-monitor]
+      networkPolicy:
```

Comment:
How come we enable here again for the hub?

Reply:
Doesn't swan-cern inherit that setting if you set it in swan?

Reply:
Yes, I set it now only in swan and the CERN specific policies in swan-cern